src/if_else/string_comparisons.md

```diff
@@ -0,0 +1,22 @@
+## Strings
+Normally to check if something is equal to something, this code would be used:
+```java
+int one = 1;
+int anotherOne = 1
+if (one == anotherOne) {
+    ...
+}
+```
+
+but for strings, it's an entirely different story, as we use `.equals` (inherited from `Object`) instead of `==`
+```java
+if (stringVariable.equals("anotherString")) {
+    ...
+}
+```
+
+Why? because for **primitive types** (such as `int`, `float`, etc), `==` compares **values**, although **for Objects** (such as `String` and `User-Defined Classes`) and Arrays, `==` compares **references**.
```

Comment:
This terminology is relatively quickly going to be inaccurate. `primitive` and `reference` types will be objects and the two kinds of `reference` types - `value` types and `identity` types will compare based on their "value" or their "identity"